set-web/src/main/java/eu/europeana/set/web/service/impl/BaseUserSetServiceImpl.java

```diff
@@ -868,6 +867,10 @@ PersistentUserSet updateUserSetForPublish(PersistentUserSet userSet,
       userSet.setCreator(creator);
     }
     userSet.setVisibility(VisibilityTypes.PUBLISHED.getJsonValue());
+    if(issued==null) {
```

Comment:
here an exception should be thrown if the value is null, the controller is responsible for setting the default values for the optional fields